bin/partitions.toit

```diff
@@ -0,0 +1,255 @@
+// Copyright (C) 2023 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
```

Comment:
Reference to a file that doesn't exist (here and other places).

Reply:
done.